core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -513,6 +513,11 @@ private function addBackendNode(): NodeDefinition
             ->getRootNode()
             ->addDefaultsIfNotSet()
             ->children()
+                ->scalarNode('route_prefix')
+                    ->info('Define the path of the Contao backend.')
+                    ->example('/admin')
+                    ->defaultValue('/contao')
```

Comment:
Should we validate for a leading slash here?

Reply:
Something like `/^\/\S*[^\/]$/` maybe? :upside_down_face:

Reply:
Validation added in 35ace90 :-)

Reply:
idk, I was just wondering. What would even happen if you defined a route (prefix) without a leading slash in Symfony? 🤔 May be it does not matter anyway?